index.html

```diff
@@ -62,8 +62,8 @@ <h3 class="box-title box--title--queue">Queue</h3>
         <div class="box-container box--container--done">
         <h3 class="box-title box--title--done">Done</h3>
         <div class="box done--box">
-            <ul class="#display-data-done-main">
-               <li class=".display-data-done"></li>
+            <ul class="display-data-done-main">
+               <li class="display-data-done"></li>
```

Comment:
For CSS classes, you don't need to put a hashtag or a dot in front of the name in the HTML. Assuming that you are trying to use classes and not IDs, I changed it so that these two class names are just the names without the dot and hashtag.